test/clt-tests/integrations/test-integrations-filebit.rec

```diff
@@ -0,0 +1,69 @@
+––– input –––
+set -b
+––– output –––
+––– input –––
+export PATH=/usr/bin:/usr/local/bin:/usr/sbin:/sbin:/bin
+––– output –––
+––– input –––
+wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | gpg --dearmor -o /usr/share/keyrings/elastic-keyring.gpg
+––– output –––
+––– input –––
+echo "deb [signed-by=/usr/share/keyrings/elastic-keyring.gpg] https://artifacts.elastic.co/packages/8.x/apt stable main" | tee -a /etc/apt/sources.list.d/elastic-8.x.list
+––– output –––
+deb [signed-by=/usr/share/keyrings/elastic-keyring.gpg] https://artifacts.elastic.co/packages/8.x/apt stable main
+––– input –––
+apt-get update > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+apt-get -y install gnupg2 openjdk-8-jre-headless curl jq filebeat > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+printf 'filebeat.inputs:\n- type: filestream\n  id: testlog\n  paths:\n    - /var/log/dpkg.log\n\noutput.elasticsearch:\n  hosts: ["http://localhost:9308"]\n  index:  "testlog_1"\n  allow_older_versions: true\n  compression_level: 0\n\nsetup.ilm:\n  enabled: false\n\nsetup.template:\n  name: "testlog_1"\n  pattern: "testlog_1"\n\nseccomp:\n  default_action: allow\n  syscalls:\n  - action: allow\n    names:\n    - rseq' > /filebeat.yml
+––– output –––
+––– block: ../base/start-searchd –––
+––– input –––
+echo "2024-06-25 10:12:34 status installed libc-bin:amd64 2.31-0ubuntu9.16\n2024-06-25 10:12:34 status installed dpkg:amd64 1.19.7ubuntu3\n2024-06-25 10:12:35 status installed base-passwd:amd64 3.5.52\n2024-06-25 10:12:35 status installed bash:amd64 5.0-7ubuntu1.2\n2024-06-25 10:12:36 status installed coreutils:amd64 8.30-3ubuntu2\n2024-06-25 10:12:36 status installed debianutils:amd64 4.11.2\n2024-06-25 10:12:36 status installed diffutils:amd64 1:3.7-5\n2024-06-25 10:12:36 status installed dpkg-dev:amd64 1.19.7ubuntu3\n2024-06-25 10:12:36 status installed libc-bin:amd64 2.31-0ubuntu9.16\n2024-06-25 10:12:37 status installed libgcc-s1:amd64 11.2.0-6ubuntu2" > dpkg.log
+––– output –––
+––– input –––
+filebeat -c /filebeat.yml -strict.perms=false -e 2>/var/log/filebeat.log & pid=$!
```

Comment:
For what reasons do we store pid there? It isn't used below anywhere

Reply:
Corrected